src/Compilers/CSharp/Portable/BoundTree/BoundTreeVisitors.cs

```diff
@@ -221,8 +221,9 @@ protected virtual bool ConvertInsufficientExecutionStackExceptionToCancelledBySt
             return true;
         }
 
+#nullable enable
```

Comment:
nit: I assume there was some trouble annotating the whole file?

Reply:
Yes, the generic Visit method is challenging to annotate.

---
In reply to: [386578401](https://github.com/dotnet/roslyn/pull/42068#discussion_r386578401) [](ancestors = 386578401)